demo/index.html

```diff
@@ -454,23 +454,46 @@
     }());
   </script>
 
-  <p>Here's an image of the line chart:</p>
-
-  <div id='line-chart-img-container'></div>
-
-  <script>
-    (function() {
-      var chart = document.getElementById('line-chart');
-      var imgContainer = document.getElementById('line-chart-img-container');
-
-      chart.addEventListener('google-chart-render', function() {
-        var img = document.createElement('img');
-        img.src = chart.getImageURI();
-        imgContainer.innerHTML = '';
-        imgContainer.appendChild(img);
+  <p>This demonstrates the ways to get the chart image URI:</p>
+
+  <dom-module id="my-el">
+    <template>
+      <p>Original chart:</p>
+      <google-chart
+          id="makeAnImgFromMe"
+          image-uri="{{chartImageUri}}"
+          data='[ ["Label", "Value"],["Memory", 80],["CPU", 55],["Network", 68] ]'></google-chart>
+      <p>Image created via upward data binding:</p>
+      <img id="imgByUpwardBinding" src$="{{chartImageUri}}">
+      <p>Image created via method return value:</p>
+      <img id="imgByEvent">
+    </template>
+    <script>
+      HTMLImports.whenReady(function () {
+        Polymer({
+          is: 'my-el',
+          behaviors: [],
+          properties: {
+            chartImageUri: {
+              type: String,
+              observer: '_chartImageUriChanged'
+            }
+          },
+          ready: function() {
+            var chart = this.$.makeAnImgFromMe;
+            var img = this.$.imgByEvent;
+            chart.addEventListener('google-chart-render', function() {
+              img.src = chart.imageURI;
```

Comment:
I see now. This is another image and you're showing both methods. For some reason I found it confusing and thought you were using the same image.